sqlalchemy_utils/types/enriched_datetime/pendulum_datetime.py

```diff
@@ -24,26 +24,18 @@ def _coerce(self, impl, value):
                 value = pendulum.from_timestamp(value)
             elif isinstance(value, str) and value.isdigit():
                 value = pendulum.from_timestamp(int(value))
-            elif isinstance(value, datetime):
-                value = pendulum.datetime(
-                    value.year,
-                    value.month,
-                    value.day,
-                    value.hour,
-                    value.minute,
-                    value.second,
-                    value.microsecond
-                )
+            elif isinstance(value, datetime.datetime):
+                value = pendulum.DateTime.instance(value)
             else:
                 value = pendulum.parse(value)
         return value
 
     def process_bind_param(self, impl, value, dialect):
         if value:
-            return self._coerce(impl, value).in_tz('UTC')
+            return self._coerce(impl, value).in_tz('UTC').naive()
         return value
 
     def process_result_value(self, impl, value, dialect):
         if value:
-            return pendulum.parse(value.isoformat())
+            return pendulum.DateTime.instance(value.replace(tzinfo=datetime.timezone.utc))
```

Comment:
flake8 is unhappy. Gonna commit this suggestion to try making it happy.

```suggestion
            return pendulum.DateTime.instance(
                value.replace(tzinfo=datetime.timezone.utc)
            )
```